src/display.rs

```diff
@@ -37,3 +37,104 @@ impl Display for DateTime {
         write!(f, "{}T{}", self.date, self.time)
     }
 }
+
+impl Display for Duration {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // P1Y2M3DT4H5M6S
```

Comment:
Unnecessary comment?

Reply:
that was just a mental crutch for me, will remove it

Reply:
removed!